src/org/traccar/api/resource/CommandResource.java

```diff
@@ -62,15 +62,19 @@ public Response send(Command entity) throws Exception {
         Context.getPermissionsManager().checkReadonly(getUserId());
         long deviceId = entity.getDeviceId();
         long id = entity.getId();
+        boolean sent;
         if (deviceId != 0 && id != 0) {
             Context.getPermissionsManager().checkPermission(Command.class, getUserId(), id);
             Context.getPermissionsManager().checkDevice(getUserId(), deviceId);
             Context.getPermissionsManager().checkUserDeviceCommand(getUserId(), deviceId, id);
-            Context.getCommandsManager().sendCommand(id, deviceId);
+            sent = Context.getCommandsManager().sendCommand(id, deviceId);
         } else {
             Context.getPermissionsManager().checkLimitCommands(getUserId());
             Context.getPermissionsManager().checkDevice(getUserId(), deviceId);
-            Context.getCommandsManager().sendCommand(entity);
+            sent = Context.getCommandsManager().sendCommand(entity, deviceId);
+        }
+        if (!sent) {
+            return Response.accepted(entity).build();
```

Comment:
I decided to notify API client that command was queued. I think code "202" - "Accepted" fits.